spec/grape/dsl/request_response_spec.rb

```diff
@@ -173,7 +173,7 @@ def self.imbue(key, value)
           it 'sets a rescue handler declared through :with option for each key in hash' do
             with_block = -> { 'hello' }
             expect(subject).to receive(:namespace_stackable).with(:rescue_handlers, StandardError => an_instance_of(Proc))
-            expect(subject).to receive(:namespace_stackable).with(:rescue_options, with: with_block)
+            expect(subject).to receive(:namespace_stackable).with(:rescue_options, {})
```

Comment:
I don't get it, why is it a problem related to the original issue you're trying to fix?